enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/SwitchToSlave.java

```diff
@@ -460,14 +458,14 @@ private void checkMyStoreIdAndMastersStoreId( StoreId myStoreId, boolean masterI
         if ( !masterIsOld )
         {
             ClusterMembers clusterMembers = resolver.resolveDependency( ClusterMembers.class );
-
             InstanceId serverId = HighAvailabilityModeSwitcher.getServerId( masterUri );
-            Predicate<ClusterMember> specification = hasInstanceId( serverId );
             Iterable<ClusterMember> members = clusterMembers.getMembers();
-            Iterable<ClusterMember> filter = filter( specification,
-                    members );
-            ClusterMember master = single(
-                    filter );
+            ClusterMember master = first( filter( hasInstanceId( serverId ), members ) );
```

Comment:
Actually I merged this too quickly. This should've been `singleOrNull`. And there's a typo in `Cannot find maser...` below.